envelope.js

```diff
@@ -54,8 +54,8 @@ module.exports = function Envelope (keystore, state) {
     if (!isEnvelope(ciphertext)) return null
 
     const envelope = Buffer.from(ciphertext.replace('.box2', ''), 'base64')
-    const feed_id = new FeedId(author).toTFK()
-    const prev_msg_id = new MsgId(previous).toTFK()
+    const feed_id = bfe.encode(author)
+    const prev_msg_id = bfe.encode(previous)
```

Comment:
:fire: do we need to check if this this is being handed `null` ?

Reply:
that is the responsiblity of envelope-js